.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,88 @@
+name: Bug or Issue Report
+description: Report a bug or issue with GEOS-Chem
+title: "Put a one-line description here"
+labels: ["category: Bug"]
+body:
+  - type: input
+    id: name
+    attributes:
+      label: Your name
+    validations:
+      required: true
+  - type: input
+    id: affiliation
+    attributes:
+      label: Your affiliation
+    validations:
+      required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened? What did you expect to happen?
+      placeholder: Describe your issue
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: What are the steps to reproduce the bug?
+      placeholder: Explain what you did
+    validations:
+      required: true
+  - type: textarea
+    id: log-files
+    attributes:
+      label: Please attach any relevant configuration and log files.
+      description: Drag and drop files like the GEOS-Chem log file, geoschem_config.rc, and HEMCO_Config.rc here. You may need to rename files to end in .txt first.
```

Comment:
Typo: `geoschem_config.rc` -> `geoschem_config.yml`

Also add `HISTORY.rc` here

Reply:
Fixed typo and added `HISTORY.rc` to the list of files to provided in commit https://github.com/geoschem/geos-chem/pull/2288/commits/e4a555f0d86ea247dfb19883ed04850304a322b4